src/controllers/auth/resendEmail/resendEmail.js

```diff
@@ -0,0 +1,32 @@
+import Users from '../../../db/models/users';
+import Verifications from '../../../db/models/verifications';
+import mail from '../../../lib/mail';
+
+const resendEmail = async (req, res) => {
+  const { email } = req.body;
+
+  if (!email) {
+    res.status(404).end();
+
+    return;
+  }
+
+  const user = await Users.findOne({ where: { email } });
+
+  if (!user) {
+    res.status(201).end();
+
+    return;
+  }
+
+  const token = await Verifications.createScopedTokenForUser(
+    user.id,
+    Verifications.SCOPE.VERIFY_EMAIL,
+  );
+
+  await mail.sendVerifyMail(email, token);
```

Comment:
这是发送一个 Email Verification 邮件，不是 发送 Verify 的邮件

```suggestion
  await mail.sendEmailVerification(email, token);
```